app/src/hvr/java/com/igalia/wolvic/PlatformActivity.java

```diff
@@ -46,14 +46,15 @@
 
 import java.util.Calendar;
 
-public class PlatformActivity extends Activity implements SurfaceHolder.Callback {
+public abstract class PlatformActivity extends Activity implements SurfaceHolder.Callback, WidgetManagerDelegate {
```

Comment:
I know we instantiate `VRBrowserActivity` but this seems unrelated to this fix, maybe we could move it to another PR, or at least a different commit

Reply:
I needed to add `WidgetManagerDelegate` so I could call `pushBackHandler()` and `popBackHandler()`, which are defined in that interface.

And then I had to make the class abstract, so I didn't actually need to implement the interface's methods (which will be implemented by the descendent class `VRBrowserActivity`).

Reply:
Fair enough.